File: WorkIt/app/src/main/java/prot3ct/workit/utils/WorkItProgressDialog.java
package prot3ct.workit.utils;


import android.app.ProgressDialog;
import android.content.Context;

public class WorkItProgressDialog {
    ProgressDialog dialog;

    public WorkItProgressDialog(Context context) {
        this.dialog = new ProgressDialog(context);
    }

    public void showProgress(String text) {
        dialog.setMessage(text);
        dialog.show();
    }

    public void dismissProgress() {
        dialog.dismiss();
    }
}

File: WorkIt/app/src/main/java/prot3ct/workit/utils/WorkItProgressDialog.kt
package prot3ct.workit.utils

import android.app.ProgressDialog
import android.content.Context

class WorkItProgressDialog(context: Context?) {
    var dialog: ProgressDialog
    fun showProgress(text: String?) {
        dialog.setMessage(text)
        dialog.show()
    }

    fun dismissProgress() {
        dialog.dismiss()
    }

    init {
        dialog = ProgressDialog(context)
    }
}